packages/builder/src/helpers/WhereQuery.ts

```diff
@@ -106,7 +106,7 @@ export const getMatch = (filters, searchableCols, whereParts = []) => {
           );
 };
 
-export const getLike = (filters, searchableCols, whereParts = []) => {
+export const getLike = (filters, _, whereParts = []) => {
```

Comment:
Why this change ?

Reply:
`searchableCols` is not used. Now `.tsconfig` says: 

```
"noFallthroughCasesInSwitch": true,
"noImplicitReturns": true,
"noUnusedLocals": true,
"noUnusedParameters": true,
```
It's not an obligation, but I thought it was in the direction of a cleaner code.